### PR TITLE
fix(Markdown): Relative <code> fontSize [TR2-440]

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

* Changes the `<code />` element relative to its container so that headings will behave as expected.

Screen Shot 2021-02-04 at 6 58 03 PM
Screen Shot 2021-02-04 at 6 58 14 PM

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: TR2-440
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.4.27-alpha.47dec8.0<br/>@codecademy/gamut-labs@9.1.1-alpha.47dec8.0<br/>@codecademy/gamut@21.2.4-alpha.47dec8.0<br/>@codecademy/styleguide@25.1.1-alpha.47dec8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601c8b43b9896f00a64c3f1e--gamut-preview.netlify.app>https://601c8b43b9896f00a64c3f1e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601c8b43b9896f00a64c3f1e>Deploy Logs</a></p>

File: packages/gamut/src/Markdown/styles/_mixins.scss [hunk 1]
```diff
@@ -93,7 +93,7 @@
     white-space: normal;
     vertical-align: baseline;
     font-family: $font-monospace;
-    font-size: 0.9rem;
+    font-size: 0.8em;
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/Markdown/styles/_mixins.scss:96]: Would you mind creating a preview/zoo deployment of this on the monolith? I suspect there are spooky changes that'll come of it.

**Review by jakemhiller — APPROVED** [review]
given that the markdown component itself overrides font-size, this seems safe

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.4.34-alpha.98bbb5.0<br/>@codecademy/gamut-labs@9.3.3-alpha.98bbb5.0<br/>@codecademy/gamut@21.3.7-alpha.98bbb5.0<br/>@codecademy/styleguide@25.2.3-alpha.98bbb5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60256837c50e295eccd8095c--gamut-preview.netlify.app>https://60256837c50e295eccd8095c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60256837c50e295eccd8095c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.4.37-alpha.c23bcd.0<br/>@codecademy/gamut-labs@10.0.3-alpha.c23bcd.0<br/>@codecademy/gamut@22.0.1-alpha.c23bcd.0<br/>@codecademy/styleguide@26.0.3-alpha.c23bcd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://602be3ecb73b6c3be02c2b14--gamut-preview.netlify.app>https://602be3ecb73b6c3be02c2b14--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/602be3ecb73b6c3be02c2b14>Deploy Logs</a></p>

**Comment by codecaaron** [comment]
Tested on preview here:  https://github.com/codecademy-engineering/Codecademy/pull/22117

Seems to be in good shape.

**Review by JoshuaKGoldberg — APPROVED** [review]
Looks great, thanks!

**Pull request merged**